packages/edit-site/src/components/global-styles/screen-variations.js

```diff
@@ -24,7 +25,11 @@ export function ScreenVariations( { name, path = '' } ) {
 	return (
 		<div className="edit-site-global-styles-screen-variations">
 			<VStack spacing={ 3 }>
-				<p>Manage style variations, saved block appearence presets.</p>
+				<p>
+					{ __( 'Customize variations of the block:' ) +
```

Comment:
We should use `sprintf` here and not concatenate strings([example](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/copy-handler/index.js#L41)).

Reply:
Good one, thanks, updating now